Tests/L0/Maven/_suite.ts

```diff
@@ -10,7 +10,7 @@ import url = require('url');
 import {Url} from 'url';
 import shell = require('shelljs');
 
-import {ToolRunner} from 'vsts-task-lib/toolrunner';
+// import {ToolRunner} from 'vsts-task-lib/toolrunner';
```

Comment:
this was busting the compilation. this isn't actually used anywhere so i just commented out the code.

i'm not sure why this compiled in the gulp build. it may be because everything was compiled together at a higher root dir or maybe a difference in gulp-tsc.